cc-snapshot

```diff
@@ -48,6 +48,27 @@ print_usage() {
   echo " cc-snapshot version 0.2.0"
 }
 
+check_size(){ 
+  if [[ "$DRY_RUN" == true ]]; then
+    echo "[DRY_RUN] would check snapshot size at $CC_SNAPSHOT_TAR_PATH"
```

Comment:
also, maybe use the same approach as on the other PR for `du`? put the check size command into its own variable, print it here if dry run is used, execute it below if not